Model/Sync/Checkout/Data.php

```diff
@@ -269,8 +269,14 @@ public function prepareLineItems($quote)
                 }
             }
         } catch (\Exception $e) {
-            $this->checkoutLogger->info('Checkout sync::prepareLineItems() - exception: ' .
-                $e->getMessage(), []);
+            $this->checkoutLogger->info(
+                __(
+                    'Failed to sync Checkout when preparing line items for sync - Checkout ID: %1, Error Message: %2',
+                    $quote->getId(),
+                    $e->getMessage()
+                ),
+                []
```

Comment:
Why the empty array here? or in the other logs as well?

Reply:
It how most of the extension logs are written, but I see that the second argument is `[]` by default. so I'm removing it.